src/components/WalletTabs.vue

```diff
@@ -93,10 +90,57 @@ const logout = async () => {
         </div>
       </main>
     </div>
+    <div
```

Comment:
Add `Position: fixed` to the mobile nav, it should be always visible. Maybe add the hide on scroll down, show on scroll up effect, not compulsory.

Reply:
it is visible all times, I changed the header and the footer from absolute/fixed to just block.
Arranged all three in columnar flex layout.
```
<HEAD>
<MAIN>
<FOOTER>
```

Reply:
If Main is longer than 100vh, the footer gets pushed out of the viewport. So the user has to scroll down till they find the footer.

Reply:
Fixed